index.js

```diff
@@ -8,7 +8,8 @@ const
     isError = require('lodash.iserror'),
     stringify = require('./lib/utils').stringify,
     debug = require('./lib/utils').debug,
-    defaultFlushTimeoutMs = 10000;
+    defaultFlushTimeoutMs = 10_000,
+    maxMessageLength = 300_000;
```

Comment:
Could you add a comment as for the reasoning for this sizing?

Reply:
964bf3203db16fcc88c8778353b5c47bbe8a586a